include/ebpf_structs.h

```diff
@@ -102,7 +102,10 @@ typedef enum
     BPF_FUNC_ktime_get_ns = 9,
     BPF_FUNC_csum_diff = 10,
     BPF_FUNC_ringbuf_output = 11,
-    BPF_FUNC_trace_printk = 12,
+    BPF_FUNC_trace_printk2 = 12,
```

Comment:
Can't we just have a single BPF_FUNC_trace_printk that accepts 5 args and have a pre-processor that back fills unused parameters with 0?

Reply:
That wouldn't catch programming bugs where the format specifiers and args don't match like
bpf_printk("Hello", arg1);
or
bpr_prink("Hello %d %d %d %d %d");